ifrs17-template/Initialization/InitSystemorphUtils.ipynb

```diff
@@ -0,0 +1,154 @@
+                "\n                                                                                        (\"TransactionalData/Actuals_ES_2020_12.csv\", ImportFormats.Actual)}}, ",
+                "\n                                                                    {\"FR\", new (string, string)[]{(\"TransactionalData/Openings_FR_2020_12.csv\", ImportFormats.Opening), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_FR_2020_12.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/Actuals_FR_2020_12.csv\", ImportFormats.Actual)}}, ",
+                "\n                                                                    {\"CH\", new (string, string)[]{(\"TransactionalData/Openings_CH_2020_12.csv\", ImportFormats.Opening), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_CH_2020_12.csv\", ImportFormats.Cashflow),  ",
+                "\n                                                                                        (\"TransactionalData/Actuals_CH_2020_12.csv\", ImportFormats.Actual), ",
+                "\n                                                                                        (\"TransactionalData/SimpleValue_CH_2020_12.csv\", ImportFormats.SimpleValue), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_CH_2020_12_MTUP10pct.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/NominalCashflows_CH_2021_3.csv\", ImportFormats.Cashflow), ",
+                "\n                                                                                        (\"TransactionalData/Actuals_CH_2021_3.csv\", ImportFormats.Actual)}}};"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Import Tasks"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static async Task<ActivityLog[]> InitializeGenericAsync(this IImportVariable import, IDataSource dataSource, string pathToImport, ",
+                "\n                                                                Dictionary<string, (string FileName, string Format)[]> filesDictionary, string[] nodes = null){",
```

Comment:
Use params instead of string[] for nodes

Reply:
Done. In this case the default value also becomes obsolete.